### ISSUE TITLE
Focus outlines are hidden on all buttons

### ISSUE THREAD

**Issue by henrycatalinismith** [henrycatalinismith/zetkin#2]
> **Note**
> Focus indicators are missing from many more places than this but I wanted to get a first draft issue written up as a starting point. Once I've had a chance to sift through everything a bit more I'll have a better idea of how best to capture the totality of the issue.

## Steps to reproduce

1. Log in and go to any page. For the sake of an example, go to the Your assignments page.
2. Use the tab key to advance keyboard focus through the page's interactive elements.

## Expected outcome

Interactive elements such as buttons should have a visible focus indicator as per [WCAG success criterion 2.4.7](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html). Not all people can use a pointer input device, so meeting this success criteria enables those people to interact with the app. Here's a GIF showing this happening on Malmö stad's website.

<img
 alt="GIF showing visible focus indicator on Malmö stad's website"
 width="512"
 src="https://user-images.githubusercontent.com/566159/191342880-f477332b-c690-467d-8de1-e6495cfdf7fa.gif">
 
## Actual outcome

There are four buttons on the Your assignments page in the following GIF, and none of them have a visible focus indicator after any of the tab keypresses denoted by the Keystroke Pro popups in the bottom.

<img
 alt="GIF showing no visible focus indicator on Your assignments page"
 width="512"
 src="https://user-images.githubusercontent.com/566159/191342920-aab7b351-5146-4894-bf4c-83957aff0892.gif">

## Severity

The success criterion covering this issue is WCAG level AA – the middle level. This is the conformance level that most organizations aim for. As this issue makes keyboard operation impossible the severity is very high.

## Recommendation

Remove all the instances of `&:focus { outline: none; }` from [`src/scss/_mixins.scss`](https://github.com/zetkin/call.zetk.in/blob/master/src/scss/_mixins.scss) and replace them with something like the following.

```scss
&:focus-visible {
    outline: 2px solid $c-brand-main;
    outline-offset: 2px;
}
```

This creates the following visible focus indicator.

<img
 alt="GIF showing visible focus indicator on Select assignment button"
 width="512"
 src="https://user-images.githubusercontent.com/566159/191344491-d8f47419-f88b-4439-8588-cc397987bec8.gif">

**Comment by henrycatalinismith** [comment]
This issue reflects an early step in the formation of my understanding of the state of affairs in the repo. I'm not sure how accurate it its any more so I'm just closing it and keeping it around as a reminder until I get around to coming up with an up-to-date version.
